docs/src/operate/lifecycle/deregister.md

```diff
@@ -0,0 +1,106 @@
+Deregistering the main device using this mechanism is not supported.
+:::
+
+Similarly, a service with topic id `device/main/service/service01` can be deregistered as follows:
+
+```sh
+mosquitto_sub --remove-retained -W 3 -t "te/device/main/service/service01/#"
+```
+
+Services of child devices can also be de-registered in a similar manner, using the child device topic id instead.
+
+:::note
+The Cumulocity mapper must be restarted for the de-registrations to take effect.
+The entities in the cloud must also be removed separately as de-registrations on the device
+are not propagated to the cloud.
+:::
+
+## Deregister a child device and its services
+
+When the topic ids of the devices and services follow the default topic scheme,
+which maintains the service topic ids (`te/device/<device-id>/service/<service-id>`)
+directly under the device topic id (`te/device/<device-id>//`) in the hierarchy,
+all the services associated to a child device can be deregistered as follows:
+
+```sh
+mosquitto_sub --remove-retained -W 3 -t "te/device/child01/service/+/#"
+```
+
+The device can also be deleted along with all of its services with a minor variation in the topic
+that includes the device as well, as follows:
+
+```sh
+mosquitto_sub --remove-retained -W 3 -t "te/device/child01/+/+/#"
+```
+
+:::note
+Deregistering parent and child entities together with a single command only works when
+the parent and children are hierarchically linked via the topics, as in the case of services
+while following the default topic scheme, which can be queried with a single topic filter.
+While using custom topic schemes, that do not maintain the same topic prefix for a device and its services,
+querying and deregistering them may not be a on-liner as in the previous example.
+:::
+
+## Deregister all child devices and services
+
+All the child devices and services can be de-registered using a wildcard topic filter that covers all the entities
+combined with a topic exclusion filter for the main device, as follows:
+
+```sh
+mosquitto_sub -v -t te/device/# -T te/device/main/#
+```
+## Unsupported cases
+
+While using the default topic scheme, the following use-cases are not supported as it's not possible to define 
+wild card topic filters for the same, as the topic scheme does not capture the parent-child relationship of devices:
+
+* Deregister all child devices of a given device
+* Deregister the entire nested child device hierarchy of a device
+
+## Custom topic schemes
+
+Most of the commands listed above are applicable only when the default topic scheme is used.
+The same rules, especially the ones involving multiple entities, do not apply to custom topic schemes,
+as different entities would be linked differently based on the topic scheme that's used.
+The filtering capabilities would also very from scheme to scheme.
+As long as you can define topic filters to select individual entities or a set of entities,
+the same can be used for de-registering as well.
```

Comment:
```suggestion
the same can be used for deregistering as well.
```

Reply:
yeah "de-registering" does seem to be the correct usage as a verb (at least according to [the search results](https://www.google.com/search?q=de-registering&oq=de-registering&gs_lcrp=EgZjaHJvbWUyCQgAEEUYORiABDIHCAEQABiABDIHCAIQABiABDIHCAMQABiABDIHCAQQABiABDIGCAUQABgeMgYIBhAAGB4yBggHEAAYHjIGCAgQABgeMgYICRAAGB7SAQgxOTA4ajBqN6gCALACAA&sourceid=chrome&ie=UTF-8))